notebooks/3_KernelMethods.ipynb

```diff
@@ -320,6 +343,9 @@
     "\n",
     "The KPCA projection may then be computed taking the first $n_{PCA}$ components \n",
     "\\begin{equation}\n",
+    "\\mathbf{T}=\\mathbf{\\hat{U}_K}\\mathbf{\\hat{\\Lambda}_K}^{1/2} = \\mathbf{\\hat{U}_K}\\mathbf{\\hat{\\Lambda}_K} (\\mathbf{\\hat{U}_K}^T\\mathbf{\\hat{U}_K}) \\mathbf{\\hat{\\Lambda}_K}^{-1/2} = \\mathbf{K} \\mathbf{\\hat{U}_K} \\mathbf{\\hat{\\Lambda}_K}^{-1/2}\n",
```

Comment:
This was already discussed in the previous notebook, and doesn't necessarily need to be enumerated again

Reply:
Yes, it was mentioned in the second notebook, but we didn't give the details of its derivation. In addition, some readers may start directly from the third notebook as suggested in the paper.
![image](https://user-images.githubusercontent.com/56018055/97764077-c0e9ba80-1b0d-11eb-8128-2032e9d91edd.png)

Reply:
Fair point